### PR TITLE
osci-4057: greenwave decision for container-images

### PR DESCRIPTION
**Pull request by Andrei-Stepanov**
Signed-off-by: Andrei Stepanov <astepano@redhat.com>

### CODE REVIEW

**Comment by Andrei-Stepanov** [comment]
@mgrabovsky please review

File: src/components/ArtifactsListByFilters.tsx [hunk 1]
```diff
@@ -47,8 +47,14 @@ import {
 import { RootStateType } from '../reducers';
 import { IStateFilters } from '../actions/types';
 import { PaginationToolbar } from './PaginationToolbar';
-import { Artifact, isArtifactMBS, isArtifactRPM } from '../artifact';
+import {
+    Artifact,
+    isArtifactMBS,
+    isArtifactRedhatContainerImage,
+    isArtifactRPM,
```
Comment (mgrabovsky) [inline src/components/ArtifactsListByFilters.tsx:54]: These three imports are unused.

Comment (Andrei-Stepanov) [reply src/components/ArtifactsListByFilters.tsx]: This is fixed.

**Comment by Andrei-Stepanov** [comment]
@mgrabovsky PR is updated according to you comments.

**Review by mgrabovsky — APPROVED** [review]
Looks good. Thanks for the changes.

**Pull request merged**